src/pages/index.js

```diff
@@ -3,8 +3,15 @@ import About from "@/components/About";
 import Projects from "@/components/Projects";
 import Timeline from "@/components/Timeline";
 import Footer from "@/components/Footer";
+import React from 'react';
+import ReactGA from 'react-ga4'; // ga4 import
 
 export default function Home() {
+
+  const TRACKING_ID = "G-4TRT9PDZQ9"; // Change tracking ID to use your own on Google Analytics. ID currently set is used for my development environment.
```

Comment:
Generated `G-9J724Z10KJ` using the `codewithaloha@gmail.com`

```suggestion
  const TRACKING_ID = "G-9J724Z10KJ";
```